snapcraft/providers.py

```diff
@@ -176,7 +176,19 @@ def get_base_configuration(
     # install snapcraft from the store's stable channel
     snap_channel = get_managed_environment_snap_channel()
     if sys.platform != "linux" and not snap_channel:
+        emit.progress(
+            "Using snapcraft from snap store channel 'latest/stable' in instance "
+            " because snap injection is only supported on Linux hosts.",
```

Comment:
minor but the full sentence might be double-spaced here

Reply:
Good catch, thanks!